lib/src/matrix.cpp

```diff
@@ -57,7 +55,7 @@ namespace konstructs {
             float ymax, xmax;
             ymax = znear * tanf(fov * PI / 360.0);
             xmax = ymax * aspect;
-            return frustum(-xmax, xmax, -ymax, ymax, znear, zfar);
+            return projection_frustum(-xmax, xmax, -ymax, ymax, znear, zfar);
```

Comment:
@nsg Apparently we have been using the frustum function from nanogui without knowing it... the end result is identical with our own version.